### PR TITLE
Auto-fav was renamed to auto-show

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3793

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-an-org-wide-team.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ Reduce channel noise by having only team owners post to the General channel. Go
 
  Reduce @mentions to keep them from overloading the entire organization. Go to the team and click **˙˙˙ More options** > **Manage Team**. On the **Settings** tab, click <strong>@mentions</strong> > turn off **Show members the option to @team or @[team name]**. 
 
-### Automatically favorite important channels
+### Automatically show important channels
 
-Favorite important channels to ensure everyone in your organization engages in specific conversations. To learn more, see [Auto-favorite channels for the whole team](https://support.office.com/article/auto-favorite-channels-for-the-whole-team-a948272c-5aa5-429c-863c-4e1e1cd6b0f6).
+Show important channels to ensure everyone in your organization engages in specific conversations. To learn more, see [Auto-favorite channels for the whole team](https://support.office.com/article/auto-favorite-channels-for-the-whole-team-a948272c-5aa5-429c-863c-4e1e1cd6b0f6). Note: This feature was recently renamed from Automatically favorite to Automatically show.
```
Comment (JohanFreelancer9) [inline Teams/create-an-org-wide-team.md]:
```suggestion
Show important channels to ensure everyone in your organization engages in specific conversations.
> [!NOTE]
> This feature was previously named Automatically favorite.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Thanks. I'll reach out to the team who owns the Auto-fav topic.

**Pull request merged**